packages/slack/slack.0.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+license: "MIT"
+maintainer: "Ahrefs <github@ahrefs.com>"
+authors: "Ahrefs <github@ahrefs.com>"
+homepage: "https://github.com/ahrefs/slack"
+bug-reports: "https://github.com/ahrefs/slack/issues"
+dev-repo: "git+https://github.com/ahrefs/slack.git"
+
+synopsis: "Slack API implementation."
+description: "OCaml interface for accessing Slack APIs and receiving events."
+
+depends: [
+  "dune"
```

Comment:
```suggestion
  "dune" {> "3.4"}
```
according to the dune-project file

Reply:
If there is a dune-project file, isn't an explicit dune dependency redundant?

Reply:
No, opam needs to know that dune has to be installed. If your opam files are dune generated, then you can omit it since dune adds it itself to the opam file, otherwise you need to manually add it with the correct lower bound